src/osd/PrimaryLogPG.h

```diff
@@ -256,6 +257,19 @@ class PrimaryLogPG : public PG, public PGBackend::Listener {
   };
   typedef std::shared_ptr<FlushOp> FlushOpRef;
 
+  struct CLSGatherOp {
+    OpContext *ctx;
+    ObjectContextRef obc;
+    OpRequestRef op;
+    std::vector<ceph_tid_t> objecter_tids;
+    int rval;
+
+    CLSGatherOp(OpContext *ctx_, ObjectContextRef obc_, OpRequestRef op_)
+      : ctx(ctx_), obc(obc_), op(op_) {}
+    ~CLSGatherOp() {}
+  };
+  typedef std::shared_ptr<CLSGatherOp> CLSGatherOpRef;
```

Comment:
using =

Reply:
@athanatos

Could you explain this comment a little more, please?
I don't quite understand what I am supposed to do.

Reply:
We generally use C++ syntax for this sort of thing.  Replace this line with:

```using CLSGatherOpRef = std::shared_ptr<CLSGatherOp>;```

However, note below that I'm not really sure why this needs to be a shared_ptr in the first place.